src/components/OSCALControlProse.js

```diff
@@ -347,8 +349,39 @@ export function OSCALReplacedProseWithParameterLabel(props) {
       </NotImplementedStatement>
     );
   }
+
+  if (props.modificationDisplay === undefined) {
+    return (
+      <Typography>
+        {props.label}
+        {prose}
+        {props.modificationDisplay}
+      </Typography>
+    );
+  }
+
+  const { controlId } = props.modificationDisplay.props;
+  const { controlPartId } = props.modificationDisplay.props;
+  const alter =
+    props.modificationDisplay.props.modificationAlters?.find(
+      (item) => item["control-id"] === controlId
+    ) ?? [];
+  const remove = alter.removes ?? [];
+  const statementIds =
+    remove.find((object) => object["by-id"] === controlPartId) ?? [];
+
+  if (statementIds.length === 0) {
+    return (
+      <Typography>
+        {props.label}
+        {prose}
+        {props.modificationDisplay}
+      </Typography>
+    );
+  }
+
   return (
-    <Typography>
+    <Typography style={{ textDecorationLine: "line-through" }}>
```

Comment:
I think it is usually better to not do `css` things in line like this. 

Maybe something like what we do in MetaData?

https://github.com/EasyDynamics/oscal-react-library/blob/83159a59b03a1c1c53359ef0d9cb2165cf05b825/src/components/OSCALMetadata.js#L40-L43

Reply:
Thanks for your feedback. I implemented by-id because I had profiles that have remove by-id, you are right that we can also remove by class, ns, ..., etc. The logic should be the same, but the only issue is to have data (profiles) with those edge cases to test the new implementation.

Reply:
@tuckerzp, I will adjust this line of code <Typography style={{ textDecorationLine: "line-through" }}> and use css as you suggested. Thanks for the example.